pkg/executor/build.go

```diff
@@ -52,6 +52,11 @@ import (
 // This is the size of an empty tar in Go
 const emptyTarSize = 1024
 
+// for testing
+var (
+	InitializeConfig = initializeConfig
```

Comment:
Please make this a local variable so only the tests in `build` package can override this.

Reply:
👍